windows-vs-16-2019.pkr.js/vs-16-2019.js.mustache

```diff
@@ -93,6 +93,13 @@
                 "./scripts/install-tipi.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "inline": [
+                "Invoke-WebRequest -Uri 'https://bootstrap.pypa.io/get-pip.py' -OutFile 'get-pip.py'",
+                "tipi run 'python get-pip.py'"
+            ]
+        },
```

Comment:
And :boom: we just introduced a bugfix in a copy which happens to be lacking in the other one.... Which is why this copy is a really bad idea.